### ISSUE TITLE
Ability to configure a custom template for "Create a New API Design" page

### ISSUE THREAD

**Issue by kavindu-perera** [Apicurio/apicurio-studio#1374]
As of now we have "Blank API", "Pet Store Example", "USPTO Dataset API" as templates to choose from. It will be better if we can create an API from "Blank API" to our needs and export it directly as a template which we can use in future for similar implementation. Doing so, we can reduce the time we would spend setting some general data.
Thank you.

**Comment by EricWittmann** [comment]
Good idea, thanks.

As a workaround right now, you could create an API with the boilerplate stuff you want and then Copy that API every time you need to create a new API (instead of creating one).

![image](https://user-images.githubusercontent.com/1890703/104477913-aad98b00-558f-11eb-8260-4bf0bae2d666.png)

**Comment by kavindu-perera** [comment]
Yes. I'm using a similar workaround for the moment where a keep separate file which I named as template.
